### PR TITLE
Dmirano/4320 mmis test scaffolding

### LINKED ISSUE #4320: [Testing] Add scaffolding for MMIS tests
### Description and related issues
We are going to need more tests for MMIS APDs soon. Create a suite for MMIS tests. Add another parallelization to the CI. Go through the existing HITECH tests and see if we need to clean anything up to make it easier to write MMIS tests. This doesn't include copying over the tests and changing them for MMIS.

### This task is done when…

- [x] MMIS test file is in place
- [x] the file runs in a separate parallel run on CircleCI
- [x] Add Cypress tests for 4209 and get enableMmis flag working with Cypress
- [x] LD Flags can be used dynamically in Cypress tests

### PR DESCRIPTION
**Pull request by mirano-darren**
Resolves #4320 

### Description
- Adds a new test suite for MMIS Cypress tests
- Adds Cypress ability to mock LD flags
- Adds Cypress tests for the create new APD page and updates affected tests
- Adds another e2e run in circle for the MMIS suite

Note: This PR is branched off of 4209, that should be merged in first before reviewing this. I'm opening up this PR to confirm circle runs the new test suite.

### Significant changes or possible side effects
All changes should be within /e2e/

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----| ----| ----| ---- |
| a MMIS APD | Doing the new create APD workflow | Page gets filled out and APD gets created | Cypress |
| a HITECH APD | Doing the new create APD workflow | Test was adapted to account for the new page | Cypress |

### Steps to manually verify this change

1. All tests on circle pass
2. Circle runs the MMIS test suite

### This pull request is ready to code review when

- [x] Automated tests are updated (and all tests are passing)
- [x] New automated test cases are documented above
- [x] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor
- [ ] Associated OpenAPI documentation has been updated
- [x] The experience passes a basic manual accessibility audit (keyboard nav,
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when

- [x] Code has been reviewed by someone other than the original author

### This pull request is ready to review when the QA has

- [ ] Verified the functionality related to the change
- [ ] Verified that the change works with Narrator on Windows
- [ ] Verified that the change works with VoiceOver on Mac
- [ ] Verified all updated pages with the WAVE tool
- [ ] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Comment by thetif** [comment]
I'll review this after Nicole's ticket is merged in since this has that code in it too

File: audit-ci.json [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+    "1085234|@cms-****/api>knex",
+    "1085234|@cms-****/e2e>@cms-****/api>knex",
+    "GHSA-4jv9-3563-23j3|@cms-****/api>knex",
+    "GHSA-4jv9-3563-23j3|@cms-****/e2e>@cms-****/api>knex"
+  ],
   "_comment": {
-    "[advisory id]": {
-      "module": "",
-      "affects": ""
+    "1085234|GHSA-4jv9-3563-23j3": {
```
Comment (tbolt) [inline audit-ci.json:11]: this may be resolved now

**Review by tbolt — APPROVED** [review]

File: audit-ci.json (hunk 1 above)
Comment (mirano-darren) [reply audit-ci.json]: hmmm seems like its not fixed, I do think we shouldn't just ignore it though.

**Review by thetif — APPROVED** [review]

**Review by jeromeleecms — APPROVED** [review]
Nothing for me to check against yet, so as long as tests pass and Ty/Tif have signed off this is good from me. 

Future state, spoke with Darren about documenting what tests are being done in human readable format so Stephanie and I can understand what is being done in Cypress + Darren said it will help keep track of tests + avoid redundancy. He will be spinning up a separate ticket for this.

**Pull request merged**